### PR TITLE
DOCSP-35990 1.4.4 release notes

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

1.4.4 release notes

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/docs-relational-migrator/DOCSP-35990/release-notes/#1.4.4-changelog

## JIRA

https://jira.mongodb.org/browse/DOCSP-35990

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65b7efd994771f7657d0908c

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/release-notes.txt [hunk 1]
```diff
@@ -10,6 +10,20 @@ Release Notes
...
+*Released January 29, 2024*
+
+Improvements:
+
+- Improved synthetic foreign keys to be preserved after schema refresh.
+- Fix for fully-offline deployments in airgapped environments. 
+- Fix for certain combinations of mappings, specifically two different
+  mappings at the same level of embedding. 
+- Various minor fixes and improvements to Sybase database support, CDC
+  stage metrics, verification stage, database exports and logging.
```
Comment (jmd-mongo) [inline source/release-notes.txt:25]: [nit]

Consider adding a comma after "database exports" like: "...CDC stage metrics, verification stage, database exports, and logging."

**Review by jmd-mongo — APPROVED** [review]
Thanks for this update, @ltran-mdb2! This one LGTM mod a nit.

Thanks,
Joe

File: source/release-notes.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/release-notes.txt]: Done!

**Comment by ltran-mdb2** [comment]
Switched the order of improvements and new feature of the 1.4.3 release notes per David Soul's request

**Pull request merged**